data/json/npcs/talk_tags.json

```diff
@@ -8,7 +8,9 @@
       "Thorazine is bad for you, you know.",
       "Thorazine is poison.",
       "You don't need thorazine, it's limiting you.",
-      "Thorazine... That's what 'they' use to keep you tame."
+      "Thorazine... That's what 'they' use to keep you tame.",
+      "Rather not cloud my mind.",
```

Comment:
<no_to_thorazine> and <yes_to_lsd> are lines tat will only appear whan an hallucination sees thorazine or lsd on the ground. They will then advise you to avoid taking thorazine (because that would make them disappear) or to take some lsd immediatly (because they live as long as the hallucination last). It's not a line that NPCs say when you offer them horazine/LSD.

Reply:
Right.